schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/history/HistoryHelper.java

```diff
@@ -68,6 +68,18 @@ private static User getUser(Key userId) {
         return null;
     }
 
+    private static Subscription getSubscription(Key subscriptionId) {
+        try {
+            if (subscriptionId != null) {
+                SubscriptionDao subscriptionDao = new SubscriptionDao();
+                return subscriptionDao.get(subscriptionId);
+            }
+        } catch (EntityNotFoundException | IllegalArgumentException e) {
+            log.warn("Failed to get Subscription", e);
```

Comment:
I've been trying to keep these more informative.  
~~~java
log.warn("Failed to get Subscription [{}]" , subscriptionId, e);
~~~